packages/react-native-client/src/ReactNativeMobileWhepClientModule.ts

```diff
@@ -78,6 +80,16 @@ export function disconnectWhepClient() {
   return nativeModule.disconnectWhep();
 }
 
+/** Pauses the WHEP stream, making the view black and disabling the sound. */
+export function pauseWhepClient() {
+  return nativeModule.pauseWhep();
+}
+
+/** Restarts the WHEP stream. Makes the view reappear along with sound. */
+export function restartWhepClient() {
```

Comment:
I'm also hesitating about "restart" here, it might be misleading. I wouldn't think it's "unpausing" the stream, rather it's resetting the stream, that is it connects again? But I'm not sure